### ISSUE TITLE
Publish to NPM

### ISSUE THREAD

**Issue by SindreSvendby** [as-ideas/oil#131]
Hi! 

We are looking into oil and think we will use it in production, but it's always hard to keep track of the latest development. Could you consider pulishing this to npm?

This way we can use [greenkeeper](https://greenkeeper.io/) or similar tool to check if we are up to date.

**Comment by CanardSauvage** [comment]
I thought about it, but in the end you still need to host all released files on a domain. The npm package would not really help and the latest version is always findable under [releases](https://github.com/as-ideas/oil/releases).

**Comment by SindreSvendby** [comment]
I'm not sure I understand. 

If this is released as an NPM package it will help us track and being up-to-date with the releases. 

What would happen internaly in our company and I assume a lot of other places is that greenkeeper will create a PR to master in our repo that have this dependency,  the PR triggers our CI that runs test and creates a new app with the updated code. In the build process  we move the files under node_modules/ to a folder where we have some of our static resources. So by enabling this option, the job will purely be a QA task to ensure that nothing has been broken. Without this, developers need to actually do something, 

create a branch, run a script to get the latest release, create a PR. in addition to check or create a method to automatically check if there is any new releases.

Hope this explains better why this will help us, and I suspect more people!

**Comment by hypery2k** [comment]
I totally agree to @SindreSvendby It helps to make dependency management within dev and deployment easier. Plus you can stats on the usage of OIL, see [here](https://www.npmjs.com/package/nativescript-fabric) for example.
